invenio_rdm_records/jsonschemas/drafts

```diff
@@ -0,0 +1 @@
+records/
```

Comment:
Should we review this? (this doesn't look like a mapping... :suspect: )

Reply:
Since they are the same... It's a symlink ;) It works like a charm and we do not need to maintain two things. Hopefully at some point converging in one. The main problem is `invenio-search` not being able to create two different indexes out of the same file since it looks for folder name 😅